### PR TITLE
Added missing add metadata source property translation [#1196]

### LINKED ISSUE #1196: Add metadata property tooltip translation is missing.
**Describe The Bug**
The translation for this button's tooltip is missing.

**To Reproduce The Bug**
Steps to reproduce the behavior. Please be as complete as possible and describe the exact steps and values used to cause the error:
1. Go to a metadata source's details page.
1. Hover the mouse over the add property button.
1. No translation is shown.

**Expected Behavior**
1. The translation should be shown.

**Screenshots**
Screen Shot 2022-03-27 at 2 55 58 PM

### PR DESCRIPTION
**Pull request by mcpierce**
Closes #1196

## Status
READY

## Does this PR contain migrations?
NO

## Before You Submit Your PR:
- [ ] Have you announced your PR on the comixed-dev mailing list?
- [ ] New feature (non-breaking change which adds functionality)
- [X] Bug fix (non-breaking change which fixes an issue)
- [ ] Refactors existing code (code changes for efficiency or maintainability)
- [ ] Security fix (be sure to include the CVE in the commit message) 
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [X] All new and existing tests passed.

### CODE REVIEW

File: comixed-webui/src/assets/i18n/fr/comic-metadata.json [hunk 1]
```diff
@@ -97,12 +98,12 @@
...
-      "select-scraping-issue": "Sélectionner ce numéro.",
-      "show-comic-details": "Afficher les détails",
-      "skip-cache": "La récupération des détails {enabled, select, true{évitera} other{utilisera}} le cache de récupération.",
-      "use-metadata-source-id": "Récupération des métadonnées en utilisant l'identifiant ComicVine."
+      "cancel-scraping": "Stop scraping comics.",
+      "reject-scraping-issue": "Reject this issue.",
+      "select-scraping-issue": "Select this issue.",
+      "show-comic-details": "Show Details",
+      "skip-cache": "Scraping will {enabled, select, true{skip} other{use}} the scraping cache.",
+      "use-metadata-source-id": "Scrape using the ComicVine id."
```
Comment (BRUCELLA2) [inline comixed-webui/src/assets/i18n/fr/comic-metadata.json]: Why change these elements ?

File: comixed-webui/src/assets/i18n/es/comic-metadata.json [hunk 2]
```diff
@@ -97,11 +98,11 @@
...
-      "cancel-scraping": "Deja de raspar cómics.",
-      "reject-scraping-issue": "Rechazar este problema.",
-      "select-scraping-issue": "Seleccione este problema.",
-      "show-comic-details": "Mostrar detalles",
-      "skip-cache": "Raspado será {enabled, select, true{skip} other{use}} la caché de raspado.",
+      "cancel-scraping": "Stop scraping comics.",
+      "reject-scraping-issue": "Reject this issue.",
+      "select-scraping-issue": "Select this issue.",
+      "show-comic-details": "Show Details",
+      "skip-cache": "Scraping will {enabled, select, true{skip} other{use}} the scraping cache.",
```
Comment (BRUCELLA2) [inline comixed-webui/src/assets/i18n/es/comic-metadata.json]: Why change these elements ?
A remark in passing, I am not sure that the Spanish translation is relevant. "Issue" in English corresponds more to "número" than to "problema" which would be closer to "problem".

**Review by BRUCELLA2 — CHANGES_REQUESTED** [review]

File: comixed-webui/src/assets/i18n/es/comic-metadata.json (hunk 2 above)
Comment (mcpierce) [reply comixed-webui/src/assets/i18n/es/comic-metadata.json]: That was a mistake on my part: I forgot that I had copied some of the existing scraping translations into the metadata file. So when I added the missing translation entries I just copied the entire file over from english to spanish. That's why these were overwritten. I'll revert this set of changes.

File: comixed-webui/src/assets/i18n/fr/comic-metadata.json (hunk 1 above)
Comment (mcpierce) [reply comixed-webui/src/assets/i18n/fr/comic-metadata.json]: See previous comment for the reason.

**Review by BRUCELLA2 — APPROVED** [review]

**Pull request merged**